src/views/Components/PlayerSettings.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import Button from '../../elements/Button'
+import arrowButton from '../../assets/svg-icons/arrow.svg';
+
+function Settings() {
+    function createPlayers() {
+        return [
+            {
+                id: 0,
+                name: "Player 1",
+            },
+            {
+                id: 1,
+                name: "Player 2",
+            },
+            {
+                id: 2,
+                name: "Player 3",
+            },
+            {
+                id: 3,
+                name: "Player 4",
+            },
+        ];
+    }
+    const [playerNames, setPlayerNames] = useState(createPlayers());
+    const [idForPlayerName, setIdForPlayerName] = useState(5);
+
+    const isThereLessThanTwelvePlayers = playerNames.length < 12;
+    const isThereMoreThanFourPlayers = playerNames.length > 4;
+    const isThisTheFirstPlayer = (i: number) => i === playerNames[0].id ? true : false;
```

Comment:
Just for practice, you don't need the `? true : false` portion of this. `i === playerNames[0].id` will already evaluate to true or false